providers/os/connection/device/linux/device_manager.go

```diff
@@ -161,7 +161,7 @@ func (d *LinuxDeviceManager) hintFSTypes(partitions []*snapshot.PartitionInfo) (
 }
 
 func (d *LinuxDeviceManager) attemptFindFstab(dir string) ([]resources.FstabEntry, error) {
-	cmd, err := d.volumeMounter.CmdRunner.RunCommand(fmt.Sprintf("find %s -type f -wholename '*/etc/fstab'", dir))
+	cmd, err := d.volumeMounter.CmdRunner().RunCommand(fmt.Sprintf("find %s -type f -wholename '*/etc/fstab'", dir))
```

Comment:
nit: can we remove the cmd runner from the vol mounter and just attach it to the device mgr struct?